doc/conf.py

```diff
@@ -61,7 +61,7 @@
     "messages/messages_introduction": "../user_guide/messages/index.html",
     "messages/messages_list": "../user_guide/messages/messages_overview.html",
     "user_guide/message-control": "messages/message_control.html",
-    "technical_reference/c_extensions": "../user_guide/c_extensions.html",
+    "technical_reference/c_extensions": "../user_guide/messages/error/no-member.html",
```

Comment:
We changed that not so long ago, I think it's find to not maintain the intermediate URL.